src/codemagic/apple/resources/beta_group.py

```diff
@@ -0,0 +1,37 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+
+from .resource import Relationship
+from .resource import Resource
+
+
+class BetaGroup(Resource):
+    """
+    https://developer.apple.com/documentation/appstoreconnectapi/prerelease_versions_and_beta_testers/beta_groups
```

Comment:
I think you messed up the docstring updates. The one which is here now belongs under respective resouce manager and vice versa.